src/index.ts

```diff
@@ -205,14 +230,28 @@ async function run() {
           children: renderSummaryTable({ diff }),
         }),
 
+        renderViolationSection({
+          title: `${String.fromCodePoint(0x274C).repeat(10)} ${diff.chunks.violations.length} ${pluralize(
```

Comment:
**curious** why the `fromCodePoint` calls here? Is there a reason we couldn't the emoji directly?